gwpy/timeseries/tests/test_timeseries.py

```diff
@@ -650,6 +650,27 @@ def test_spectrogram2(self, losc):
         # note: bizarre stride length because 4096/100 gets rounded
         assert sg.dt == 0.010009765625 * units.second
 
+    @utils.skip_minimum_version('scipy', '0.16')
+    def test_fftgram(self, losc):
+        fgram = losc.fftgram(1)
+        fs = int(losc.sample_rate.value)
+        [f, t, sxx] = signal.spectrogram(losc, fs,
+                                         window='hann',
+                                         nperseg=fs,
+                                         mode='complex')
+        numpy.testing.assert_array_equal(losc.t0.value + t, fgram.xindex.value)
```

Comment:
`assert_array_equal` can be access as `utils.assert_array_equal`, which may make things a little cleaner.